.travis.yml

```diff
@@ -78,17 +77,12 @@ before_install:
 # Install PyTorch for Linux with no CUDA support
 - |
   if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-    if [ "${PYTHON}" = "3.6" ]; then
-      pip install https://download.pytorch.org/whl/cpu/torch-1.1.0-cp36-cp36m-linux_x86_64.whl
-    fi
-    if [ "${PYTHON}" = "3.7" ]; then
-      pip install https://download.pytorch.org/whl/cpu/torch-1.1.0-cp37-cp37m-linux_x86_64.whl
-    fi
+    pip install -q torch==1.3.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
   fi
 
 install:
 # Install fonduer dev deps and check code style
-- make -s dev_extra
+- make -s dev_extra > /dev/null
```

Comment:
Please do not silent this line.
Without knowing the version of each dependent package, debugging would become difficult.
For example, it might be harder to find networkx 2.4 was installed here https://github.com/HazyResearch/fonduer/pull/335#issuecomment-543381163

Reply:
Agree! This is hacky to reduce the log size. Any suggestion there?

Reply:
I'd suggest to just use `make dev_extra` here (`-s` option does not seem to have any effect here).
If you worry about the log size, #311 produces way more logs because snorkel-metal is not compatible (more precisely, snorkel-metal is using deprecated APIs of torch) with torch>=1.2. That's why #319 pins torch on 1.1.0.